src/Makefile.am

```diff
@@ -43,6 +44,27 @@ redshift_LDADD += \
 	$(DRM_LIBS) $(DRM_CFLAGS)
 endif
 
+if ENABLE_WAYLAND
+redshift_SOURCES += gamma-wl.c gamma-wl.h
+
+AM_CFLAGS += $(WAYLAND_CFLAGS)
+
+redshift_LDADD += \
+	$(WAYLAND_LIBS) $(WAYLAND_CFLAGS)
+
+nodist_redshift_SOURCES = \
+	gamma-control-client-protocol.h \
+	gamma-control-protocol.c \
+	orbital-authorizer-protocol.c \
+	orbital-authorizer-client-protocol.h
+
+BUILT_SOURCES = \
```

Comment:
Seems that `nodist_redshift_SOURCES` and `BUILT_SOURCES` are duplicit. Can we remove one of them?

Reply:
We need both variables, but populate them from a common variable